test/libdnf/iniparser/test_iniparser.cpp

```diff
@@ -127,15 +127,15 @@ void IniparserTest::test_iniparser2() {
 # Test comment2
 [section1]
 key1 = value1
-key2 = two line  
+key2 = two line
```

Comment:
Regarding this test. It's throwing errors for obvious reasons. I can either normalize the string to not trigger linters but then the assertions will have to be changed as well.

Reply:
I would exclude the tests that involve text parsing directly by instructing pre-commit to not perform the checks on such files

Reply:
ok. That should let the checks pass.

Reply:
all is passing locally now.waiting for the build and tests to be green in GH